Screens/LandingPage.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { View, Text, StyleSheet, Button } from 'react-native';
+
+
+const LandingPage = ({navigation}) => {
+
+    return (
+        <View style= {styles.container}>
+        
+            <Text style={styles.title}>
+                LiveBeat
+            </Text>
+            
+            <Text style={styles.description}>
```

Comment:
Instead of `styles.description`, we should use `styles.text`

```suggestion
            <Text style={styles.text}>
```